### ISSUE TITLE
Can we include an "interrupt" button for the simulations?  

### ISSUE THREAD

**Issue by Kenkleinman** [katieleap/normanal#21]
Did we talk about this already?

**Comment by katieleap** [comment]
Yes, we did talk about it, I've been working on it for a few months, and the (current) answer is no: [see this post](https://groups.google.com/d/msg/shiny-discuss/j9-NeG9v5ds/O5PDlSaMAgAJ). 

I can interrupt the simulation in an interactive R session, but there's not a way to communicate with the command line from shiny as of right now (and I don't think that's a desirable feature) so that solution doesn't work. Additionally, reactive values are set when they're called as variables in a function in R, so a solution wherein an action button is checked to see if the process has been told to interrupt doesn't work because it only checks on the function call, not during iterations. Using a child or parallel process is an option, but not necessarily a good one. I think it's best to wait for a feature from shiny and focus on the other approach to this problem in issue #8
